.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+on:
+  push:
+    branches:
+    - "master"
+  pull_request:
+    branches:
+    - "master"
+
+name: Run Tox tests
+
+jobs:
+  tox_test:
+    name: Tox test
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
The Actions in you fork report:

> Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3.

Can we use v4 here to avoid the warning?

Reply:
Done.